bin/vc-js

```diff
@@ -148,6 +148,7 @@ program
       const result = await vc.issue({
         credential,
         suite: testSuite,
+        controller: assertionController,
```

Comment:
I don't know why this new property is needed.

Reply:
Removed.